R/check_frontmatter.R

```diff
@@ -55,7 +55,8 @@ check_frontmatter <- function(
 
   # checks common to all protocol types
   yml_string <- list("title" = yml_protocol$title,
-                  "file_manager" = yml_protocol$file_manager)
+                  "file_manager" = yml_protocol$file_manager,
+                  "bibliography" = yml_protocol$bibliography)
```

Comment:
```suggestion
                  "file_manager" = yml_protocol$file_manager)
```

The `bibliography` field can also be a character vector, instead of a string (= a length one character vector). Multiple bib files are often useful, for instance when [citing R and R packages](https://inbo.github.io/tutorials/tutorials/r_citations_markdown/#citing-r-and-r-packages)

Reply:
But indeed, a check for the bibliography field was missing. I think a simple is.character() check will suffice here. Can you add one?

In `check_structure` a more elaborate check is already included which checks if bib files mentioned in bibliography fields are present in the protocol folder.